### DISCUSSION TITLE
Render Layers with USD

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/maya-usd#1819] [Q&A]
Hi everyone,

Has anyone had any success with adding USD prims to render layers with the Render Setup tool in Maya? It seems that the Render Setup tool does not understand USD prims and can only recognize and add the USD top level transform and shape node.

Does anyone have any solution for this issue right now, or any suggestions for achieving the same workflow with USD in Maya?
 
As a workaround we are using Arnold for our render engine and it seems that Arnold Operators can recognize USD prims by name. So we can hook up these operators into the Render Setup tool.

**Comment by Aquamenti9** [comment]
Hello,

Our team have the same issue regarding render layers, we can't find a way to use USD with the Render Setup.
Also in Maya 2024 Arnold Standin doesn't seem to recognize USD hierachy anymore, so sadly the workaround is not an option.
Has there been some progress on this subject? It's a big problem to not be able to render USD properly.

Thanks!
